diff --git a/aces_1.2/config.ocio b/aces_1.2/config.ocio
index 1e865d6..db92602 100644
--- a/aces_1.2/config.ocio
+++ b/aces_1.2/config.ocio
@@ -1,6 +1,6 @@
 ocio_profile_version: 1
 
-search_path: luts
+search_path: "luts:aces_vwg_gm"
 strictparsing: true
 luma: [0.2126, 0.7152, 0.0722]
 
@@ -22,6 +22,12 @@ roles:
 
 displays:
   ACES:
+    - !<View> {name: ACES (Rec.709) - VWG, colorspace: Output - ACES - Rec.709 - ACES GM VWG}
+    - !<View> {name: ARRI ALF-2 (Rec.709) - VWG, colorspace: Output - ARRI ALF-2 - Rec.709 - ACES GM VWG}
+    - !<View> {name: ARRI K1S1 (Rec.709) - VWG, colorspace: Output - ARRI K1S1 - Rec.709 - ACES GM VWG}
+    - !<View> {name: RED IPP2 (Rec.709) - VWG, colorspace: Output - RED IPP2 - Rec.709 - ACES GM VWG}
+    - !<View> {name: Filmlight TCAM (Rec.709) - VWG, colorspace: Output - Filmlight TCAM - Rec.709 - ACES GM VWG}
+    - !<View> {name: -------------------------------------, colorspace: Utility - Raw}
     - !<View> {name: sRGB, colorspace: Output - sRGB}
     - !<View> {name: DCDM, colorspace: Output - DCDM}
     - !<View> {name: DCDM P3D60 Limited, colorspace: Output - DCDM (P3D60 Limited)}
@@ -50,7 +56,7 @@ displays:
     - !<View> {name: Log, colorspace: Input - ADX - ADX10}
 
 active_displays: [ACES]
-active_views: [sRGB, DCDM, DCDM P3D60 Limited, DCDM P3D65 Limited, P3-D60, P3-D65 ST2084 1000 nits, P3-D65 ST2084 2000 nits, P3-D65 ST2084 4000 nits, P3-DCI D60 simulation, P3-DCI D65 simulation, P3D65, P3D65 D60 simulation, P3D65 Rec.709 Limited, P3D65 ST2084 108 nits, Rec.2020, Rec.2020 P3D65 Limited, Rec.2020 Rec.709 Limited, Rec.2020 HLG 1000 nits, Rec.2020 ST2084 1000 nits, Rec.2020 ST2084 2000 nits, Rec.2020 ST2084 4000 nits, Rec.709, Rec.709 D60 sim., sRGB D60 sim., Raw, Log]
+active_views: [ACES (Rec.709) - VWG, ARRI ALF-2 (Rec.709) - VWG, ARRI K1S1 (Rec.709) - VWG, RED IPP2 (Rec.709) - VWG, Filmlight TCAM (Rec.709) - VWG, -------------------------------------, sRGB, DCDM, DCDM P3D60 Limited, DCDM P3D65 Limited, P3-D60, P3-D65 ST2084 1000 nits, P3-D65 ST2084 2000 nits, P3-D65 ST2084 4000 nits, P3-DCI D60 simulation, P3-DCI D65 simulation, P3D65, P3D65 D60 simulation, P3D65 Rec.709 Limited, P3D65 ST2084 108 nits, Rec.2020, Rec.2020 P3D65 Limited, Rec.2020 Rec.709 Limited, Rec.2020 HLG 1000 nits, Rec.2020 ST2084 1000 nits, Rec.2020 ST2084 2000 nits, Rec.2020 ST2084 4000 nits, Rec.709, Rec.709 D60 sim., sRGB D60 sim., Raw, Log]
 
 colorspaces:
   - !<ColorSpace>
@@ -4964,3 +4970,78 @@ colorspaces:
     allocation: lg2
     allocationvars: [-8, 5, 0.00390625]
     to_reference: !<ColorSpaceTransform> {src: ACES - ACEScg, dst: ACES - ACES2065-1}
+
+  - !<ColorSpace>
+    name: Output - ACES - Rec.709 - ACES GM VWG
+    family: Output
+    equalitygroup: ""
+    bitdepth: 32f
+    description: |
+      ACES Gamut Mapping Virtual Working Group - ACES (Rec.709) Output Transform
+    isdata: false
+    allocation: uniform
+    allocationvars: [0, 1]
+    from_reference: !<GroupTransform>
+      children:
+        - !<ColorSpaceTransform> {src: ACES - ACES2065-1, dst: ACES - ACEScg}
+        - !<FileTransform> {src: ACEScg_to_ACES_Rec709.csp, interpolation: tetrahedral}
+
+  - !<ColorSpace>
+    name: Output - ARRI ALF-2 - Rec.709 - ACES GM VWG
+    family: Output
+    equalitygroup: ""
+    bitdepth: 32f
+    description: |
+      ACES Gamut Mapping Virtual Working Group - ALF-2 (Rec.709) Output Transform
+    isdata: false
+    allocation: uniform
+    allocationvars: [0, 1]
+    from_reference: !<GroupTransform>
+      children:
+        - !<ColorSpaceTransform> {src: ACES - ACES2065-1, dst: ACES - ACEScg}
+        - !<FileTransform> {src: ACEScg_to_ALF-2_Rec709.csp, interpolation: tetrahedral}
+
+  - !<ColorSpace>
+    name: Output - ARRI K1S1 - Rec.709 - ACES GM VWG
+    family: Output
+    equalitygroup: ""
+    bitdepth: 32f
+    description: |
+      ACES Gamut Mapping Virtual Working Group - ARRI K1S1 (Rec.709) Output Transform
+    isdata: false
+    allocation: uniform
+    allocationvars: [0, 1]
+    from_reference: !<GroupTransform>
+      children:
+        - !<ColorSpaceTransform> {src: ACES - ACES2065-1, dst: ACES - ACEScg}
+        - !<FileTransform> {src: ACEScg_to_ARRI_K1S1.csp, interpolation: tetrahedral}
+
+  - !<ColorSpace>
+    name: Output - RED IPP2 - Rec.709 - ACES GM VWG
+    family: Output
+    equalitygroup: ""
+    bitdepth: 32f
+    description: |
+      ACES Gamut Mapping Virtual Working Group - RED IPP2 (Rec.709) Output Transform
+    isdata: false
+    allocation: uniform
+    allocationvars: [0, 1]
+    from_reference: !<GroupTransform>
+      children:
+        - !<ColorSpaceTransform> {src: ACES - ACES2065-1, dst: ACES - ACEScg}
+        - !<FileTransform> {src: ACEScg_to_IPP2_Rec709.csp, interpolation: tetrahedral}
+
+  - !<ColorSpace>
+    name: Output - Filmlight TCAM - Rec.709 - ACES GM VWG
+    family: Output
+    equalitygroup: ""
+    bitdepth: 32f
+    description: |
+      ACES Gamut Mapping Virtual Working Group - Filmlight TCAM (Rec.709) Output Transform
+    isdata: false
+    allocation: uniform
+    allocationvars: [0, 1]
+    from_reference: !<GroupTransform>
+      children:
+        - !<ColorSpaceTransform> {src: ACES - ACES2065-1, dst: ACES - ACEScg}
+        - !<FileTransform> {src: ACEScg_to_TCAM_Rec709.csp, interpolation: tetrahedral}
